libs/color-sensor/color.ts

```diff
@@ -179,6 +183,28 @@ namespace sensors {
             return this.getNumber(NumberFormat.UInt8LE, 0)
         }
 
+        /**
+         * Get the current rgb values from the color sensor.
+         * @param sensor the color sensor to query the request
+         */
+        //% help=sensors/color-sensor/rgbraw
+        //% block="**color sensor** %this| rgbraw"
```

Comment:
In my opinon this is advanced and we don't add blocks for advanced APIs. Make this a JavaScript only api.

Reply:
Sorry, did not quite understand. I think that RGB is very necessary for solving problems of determining color in rgb and translating through this mode in hsv. For competition, we need an RGB mode.